drivers/i2c/target/eeprom_target.c

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#define DT_DRV_COMPAT atmel_at24
+#define DT_DRV_COMPAT zephyr_target_eeprom
```

Comment:
Perhaps this should include "i2c" somehow? We could add support for other EEPROM-like targets on other busses (e.g. SPI) later on.

Reply:
Do you mean something like `zephyr_target_i2c_eeprom`?

Reply:
Either that or `zephyr,i2c-target-eeprom`. We may also end up adding more I2C targets down the line (which would then become `zephyr,i2c-target-foo`).

Reply:
Done, I wasn't sure if I should rename the driver file as well.